src/OpenLoco/src/Main.cpp

```diff
@@ -5,22 +5,50 @@
 #define WIN32_LEAN_AND_MEAN
 #endif
 #include <windows.h>
+
+#include <shellapi.h>
 /**
  * The function that is called directly from the host application (loco.exe)'s WinMain. This will be removed when OpenLoco can
  * be built as a stand alone application.
  */
 // Hack to trick mingw into thinking we forward-declared this function.
 extern "C" __declspec(dllexport) int StartOpenLoco(HINSTANCE hInstance, HINSTANCE hPrevInstance, LPSTR lpCmdLine, int nCmdShow);
-extern "C" __declspec(dllexport) int StartOpenLoco([[maybe_unused]] HINSTANCE hInstance, [[maybe_unused]] HINSTANCE hPrevInstance, LPSTR lpCmdLine, [[maybe_unused]] int nCmdShow)
+extern "C" __declspec(dllexport) int StartOpenLoco([[maybe_unused]] HINSTANCE hInstance, [[maybe_unused]] HINSTANCE hPrevInstance, [[maybe_unused]] LPSTR lpCmdLine, [[maybe_unused]] int nCmdShow)
 {
-    return OpenLoco::main(lpCmdLine);
+    // We need to get the Wide command line and convert it to utf8 command line parameters
+    const auto cmdline = GetCommandLineW();
+
+    int argc{};
+    auto* argw = CommandLineToArgvW(cmdline, &argc);
+
+    // This will hold a utf8 string of the command line
+    std::vector<std::string> argvStrs;
+    argvStrs.resize(argc);
+
+    for (auto i = 0; i < argc; ++i)
+    {
+        int length = WideCharToMultiByte(CP_UTF8, 0, argw[i], -1, 0, 0, NULL, NULL);
```

Comment:
When WideCharToMultiByte returns 0 it means it failed, right now it would just keep the string empty. We should probably check for this and at least print an error into the log.

Reply:
This is prior to initialization of the log mind.

Reply:
Before its initialized it defaults to printing to stdout/stderr so thats fine, it won't be in the log file but we'll see it in the terminal.

Reply:
okay done.